### DISCUSSION TITLE
Can I use milvus to cluster?

### DISCUSSION THREAD

**Discussion by shawn0wang** [milvus-io/milvus#27604] [Q&A and General discussion] [answered]
If yes, plz show me demo

**Comment by xiaofan-luan** [comment]
https://milvus.io/docs/install_cluster-milvusoperator.md

**Reply by shawn0wang** [reply]
I mean K-means clustering or something else

**Reply by filip-halt** [reply, marked as answer]
At the moment not really. We don't build an index across all the data but rather split data into 512mb chunks and create indexes per chunk. This means we have many separate kmeans clusters per dataset.
